lib/createWorkers.js

```diff
@@ -48,10 +50,37 @@ function createWorkers(handlers, config, options = {}) {
 
   return {
     listen: co.wrap(_listen),
-    close: co.wrap(_close)
+    close: co.wrap(_close),
+    get emitter() {
+      return emitter;
+    },
+    wait(key) {
```

Comment:
Ok, to sum up the discussion for everybody, the timeout actually has a usecase: with a default value < mocha's default tiemout, it will let us knwo where the test hanged, which is quite convenient, so I will implement it.